### PR TITLE
Div tab not closed

### PR DESCRIPTION
**Pull request by pratikshasaman**
fix: Div tag not closed

### CODE REVIEW

File: inc/widgets-manager/widgets/class-cart.php [hunk 1]
```diff
@@ -592,7 +592,8 @@ protected function render() {
 									<span class="elementor-screen-only">Cart</span>
```
Comment (vrundakansara) [inline inc/widgets-manager/widgets/class-cart.php:592]: Add translations to these strings.

Comment (SuryawanshiPrajakta) [reply inc/widgets-manager/widgets/class-cart.php]: Done!

File: inc/widgets-manager/widgets/class-cart.php [hunk 2]
```diff
@@ -589,10 +589,11 @@ protected function render() {
 								</span>
 								<span class="elementor-button-icon" data-counter="<?php echo ( ( empty( $woocommerce->cart ) ) ? 0 : $woocommerce->cart->cart_contents_count ); ?>">
 									<i class="eicon" aria-hidden="true"></i>
-									<span class="elementor-screen-only">Cart</span>
+									<span class="elementor-screen-only"><?php __( 'Cart', 'header-footer-elementor' ); ?></span>
```
Comment (vrundakansara) [inline inc/widgets-manager/widgets/class-cart.php:592]: This will not echo the string.
Use _e()

Comment (SuryawanshiPrajakta) [reply inc/widgets-manager/widgets/class-cart.php]: Done!

**Pull request merged**